packages/shared/lib/core/wallet/stores/selected-account-assets.store.ts

```diff
@@ -27,9 +26,9 @@ export const assetFilter: Writable<AssetFilter> = writable({
 })
 
 export const selectedAccountAssets: Readable<IAccountAssets> = derived(
-    [activeProfileId, selectedAccountId, persistedAssets],
-    ([$activeProfileId, $selectedAccountId]) => {
-        if ($activeProfileId && $selectedAccountId) {
+    [activeProfileId, persistedAssets],
```

Comment:
i think we need here the `selectedAccountId` because it needs to be refreshed when that changes

Reply:
I tested it and the store is reactive even if I don't add `selectedAccountId`.